Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1669,7 +1675,7 @@ namespace PlayerCommands
 			}
 		}
 
-		int pages = (matchingItems / 40) + 1;
+		int pages = (matchingItems / 30) + 1;

```

Comment:
30 looks like a magical number. Perhaps to rename to what is it in reality
int ItemsPerPage = 30;
and to use the variable in the formulas?